### PR TITLE
File added: Assignment 6 part2, 7 and 8

### PR DESCRIPTION
**Pull request by shivani-pawbake**

### CODE REVIEW

**Review by pooja-nagare — APPROVED** [review]

**Review by SnehaMakade — APPROVED** [review]

File: Shivani/HTML/assign6/ques.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.code3 {
+    margin-right: 15%;
+}
+.lc {
+    margin-right: 128px;
+}
+.ls {
+    margin-right: 35px;
+}
+.lk {
```
Comment (shivling123) [inline Shivani/HTML/assign6/ques.css]: give proper name to all class
which can identify easly

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes and push asap on git

File: Shivani/HTML/assign7/login.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>login</title>
+        <link rel="stylesheet" href="style.css" />
+
```
Comment (shivling123) [inline Shivani/HTML/assign7/login.html]: remove space

File: Shivani/HTML/assign7/login.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        <link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaina+2&display=swap" rel="stylesheet" />
+    </head>
+    <body>
+        <div class="login-form">
+            <h1>Login Form</h1>
+            <hr />
+            <form action="#" method="POST">
+                <label for="user"><h3>User Name</h3></label>
+                <input type="text" name="user" placeholder="User Name" /><br />
```
Comment (shivling123) [inline Shivani/HTML/assign7/login.html]: give id to input field

File: Shivani/HTML/assign7/personal.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                <div class="block1">
+                    <h1>Job Portel</h1>
+                </div>
+                <hr />
+                <div class="block2">
+                    <h2>Personal details</h2>
+                </div>
+                <hr />
+                <div class="block4">
+                    <label>First Name:</label>
```
Comment (shivling123) [inline Shivani/HTML/assign7/personal.html]: give id to input filed and for attribute to label it must be same both

File: Shivani/HTML/assign7/personal.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+                    <label for="age"> Skill Set:</label>
+                    <input type="checkbox" id="HTML" name="skillset" value="HTML" />
+                    <label for="vehicle1">HTML</label>
+                    <input type="checkbox" id="Javascript" name="skillset" value="Javascript" />
+                    <label for="vehicle2">Javascript</label>
+                    <input type="checkbox" id="java" name="skillset" value="java" />
+                    <label for="vehicle3">Java</label>
+                    <input type="checkbox" id="oracle" name="skillset" value="oracle" />
+                    <label for="vehicle3">oracle</label><br />
+                    <label for="highest Qualification">Highest Qualification:</label>
```
Comment (shivling123) [inline Shivani/HTML/assign7/personal.html:65]: give id to input filed and for attribute to label it must be same both

File: Shivani/HTML/assign7/profession.html [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+            </ul>
+        </header>
+        <div class="block6">
+            <div class="block4">
+                <h1>Professional Details</h1>
+            </div>
+            <hr />
+            <div class="block5">
+                <h3>Work Experience</h3>
+                <hr />
```
Comment (shivling123) [inline Shivani/HTML/assign7/profession.html:27]: give id to input field
in all code make this change

**Review by shivling123 — CHANGES_REQUESTED** [review]
do changes as per required and push file asap

**Review by pritikekan — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request closed without merging**